spec/forms/candidate_interface/restructured_work_history/job_form_spec.rb

```diff
@@ -39,15 +39,15 @@
     it { is_expected.to validate_length_of(:role).is_at_most(60) }
     it { is_expected.to validate_length_of(:organisation).is_at_most(60) }
 
-    include_examples 'validation for a start date', 'restructured_work_history/job_form', verify_presence: true
```

Comment:
why is ths presence check skipped - as it's not reflected on the form, so I expect that we still want to verify it through the specs.

Reply:
Removed as part of https://github.com/DFE-Digital/apply-for-teacher-training/pull/4197/commits/bea0b6a5bc5c60f1c335e029f946d2644cfba0ab

Rationale - we don't (AFAICT) have any test cases where we don't verify presence, making an option redundant. Can drop this commit altogether if we want to retain the option.